src/server.cpp

```diff
@@ -1389,20 +1400,22 @@ void CServer::ProcessData ( const CVector<CVector<int16_t> >& vecvecsData,
                     {
                         // left channel
                         vecsOutData[k] = Double2Short (
-                            vecsOutData[k] + vecsData[i] * dGain );
+                                    vecsOutData[k] + vecsData[i] * dGain * dPanCoefL);
```

Comment:
Maybe instead of having a dGain and a dPan we should combine both and just multiply each sample with that single gain value, this would safe us some multiplications.

Reply:
Yes, to have dGainL and dGainR computer once by chaneel (in case of stereo) and not by every sample is better.